### DISCUSSION TITLE
FloatButton如何使用disabled属性

### DISCUSSION THREAD

**Discussion by yaxiyeye** [ant-design/ant-design#41016] [Q&A]
我想在FloatButton中使用disabled属性应该如何去做，点击事件渲染之后好像是绑定在div上的，不是渲染的button上

**Comment by MadCcc** [comment]
暂时不支持 `disabled` 属性，可以通过修改 `pointer-events` 等方法先 workaround。

**Reply by yaxiyeye** [reply]
好的大佬，已经解决
